brjs-core/src/main/java/org/bladerunnerjs/model/NodeImporter.java

```diff
@@ -205,22 +205,30 @@ private static void findAndReplaceInTextFiles(BRJS brjs, Collection<File> files,
 	{
 		for (File f : files) {
 			if (f.length() != 0) {
-				if (checkFileMimeType(f).startsWith("text")) {
-					findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+				try {
+					if (checkFileMimeType(f).startsWith("text")) {
+						findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+					}
+				}
+				catch (MagicMatchNotFoundException e) {
+					continue;
 				}
 			}
 		}
 	}
 
-	private static String checkFileMimeType(File file) throws IOException {
+	private static String checkFileMimeType(File file) throws IOException, MagicMatchNotFoundException {
 		byte[] data = Files.readAllBytes(Paths.get(file.getAbsolutePath()));
```

Comment:
The use of MimeMagic was the cause of #1315 since files with `£` etc. aren't seen as text files. The root cause of the need for MimeMagic was a String == String comparison which meant we always wrote to the file even if we hadn't matched any namespaces to change. This has now been fixed so we can remove the need for MimeMagic and do an initial check for known file types, to prevent reading extra image files we know won't change, and then only write to a file where we need to change the contents.